sync-core/src/main/java/com/cloudant/sync/query/IndexManager.java

```diff
@@ -94,18 +93,32 @@ public IndexManager(Datastore datastore) {
         queue = Executors.newSingleThreadExecutor();
 
         final String filename = datastore.extensionDataFolder(EXTENSION_NAME) + File.separator
-                                                                        + "indexes.sqlite";
+                                                                              + "indexes.sqlite";
         SQLDatabase sqlDatabase = null;
         try {
             sqlDatabase = queue.submit(new Callable<SQLDatabase>() {
```

Comment:
After our Slack-chat I now remember why we did not refactor the IndexManager back when we refactored the rest of the project to use SQLDatabaseQueue.  I opened a separate FB ticket https://cloudant.fogbugz.com/f/cases/46120 to handle the refactoring for the IndexManager.